Gem/Code/Source/Components/Multiplayer/EnergyBallComponent.h

```diff
@@ -25,17 +23,26 @@ namespace MultiplayerSample
         void OnActivate(Multiplayer::EntityIsMigrating entityIsMigrating) override;
         void OnDeactivate(Multiplayer::EntityIsMigrating entityIsMigrating) override;
 
-        void OnTick(float deltaTime, AZ::ScriptTimePoint time) override;
 #if AZ_TRAIT_CLIENT
         void HandleRPC_BallExplosion(AzNetworking::IConnection* invokingConnection, const HitEvent& hitEvent) override;
 #endif
 
     private:
 #if AZ_TRAIT_CLIENT
         void DebugDraw();
+        void OnBallActiveChanged(bool active);
+
+        AZ::ScheduledEvent m_debugDrawEvent{ [this]()
+        {
+            DebugDraw();
+        }, AZ::Name("EnergyBallDebugDraw") };
+
+        AZ::Event<bool>::Handler m_ballActiveHandler{ [this](bool active)
+        {
+            OnBallActiveChanged(active);
+        } };
```

Comment:
out of curiosity is there some format that lets you forgo the lambda and do something like
```
AZ::Event<bool>::Handler m_ballActiveHandler = &EnergyBallComponent::OnBallActiveChanged;
```
?

Reply:
Sort of? 

I think I'd have to make it a static method instead of a class method and make it figure out which instance is affected to work around the lack of a "this" pointer, so the lambda seems like a tidier solution unless you know a trick I'm overlooking.